storage_integration_test.go

```diff
@@ -0,0 +1,115 @@
+// +build integration
+
+package tprstorage
+
+/*
+	Usage:
+
+		go test -tags=integration $(glide novendor) [FLAGS]
+
+	Flags:
+
+		-integration.ca string
+			CA file path (default "$HOME/.minikube/ca.crt")
+		-integration.crt string
+			certificate file path (default "$HOME/.minikube/apiserver.crt")
+		-integration.key string
+			key file path (default "$HOME/.minikube/apiserver.key")
+		-integration.server string
+			Kubernetes API server address (default "https://192.168.64.16:8443")
+*/
+
+import (
+	"context"
+	"flag"
+	"os/user"
+	"path"
+	"testing"
+
+	"github.com/giantswarm/micrologger"
+	"github.com/giantswarm/microstorage/storagetest"
+
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+)
+
+var (
+	server  string
+	crtFile string
+	keyFile string
+	caFile  string
+)
+
+func init() {
+	u, err := user.Current()
+	homePath := func(relativePath string) string {
+		if err != nil {
+			return ""
+		}
+		return path.Join(u.HomeDir, relativePath)
+	}
+
+	flag.StringVar(&server, "integration.server", "https://192.168.64.16:8443", "Kubernetes API server address")
```

Comment:
I can imagine it would be a different IP for me and on minikube not everybody will have secure connections. I think there should be a more simply default.